src/main/java/duke/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+/**
+ * Represent command that user can input.
+ */
+public abstract class Command {
+
+    private boolean isExit = false;
```

Comment:
Can this name be more descriptive?

Reply:
Hmm. I took isExit from the CS2103T website though?